hm_30/hm_030.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import random
+import json
+
+GOOGLE = "https://www.google.com/"
+FACEBOOK = "https://www.facebook.com/"
+TWITTER = "https://twitter.com/"
+AMAZON = "https://www.amazon.com/"
+APPLE = "https://www.apple.com/"
+ALL = [GOOGLE, FACEBOOK, TWITTER, AMAZON, APPLE]
+while True:
+    for item in ALL:
+        res = requests.get(random.choice(ALL))
+        print(f"url:{item}, status_code:{res.status_code}, html_length:{len(res.text)}")
+
+import requests
+
+while True:
```

Comment:
You don't need an infinite loop here. It is enough to ask a user for an input and show a result only once